election.html

```diff
@@ -0,0 +1,289 @@
+<!doctype html>
+<html lang="en_NZ">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Vote with IRMA</title>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+    <link type="text/css" rel="stylesheet" href="style-election.css">
+
+    <!-- development version, includes helpful console warnings -->
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
```

Comment:
So how were firsts steps with Vue.js?

Reply:
I found it much easier to start than mithril :D